chunky/src/java/se/llbit/chunky/ui/render/tabs/WaterTab.java

```diff
@@ -185,6 +188,7 @@ public void initialize(URL location, ResourceBundle resources) {
       scene.setWaterPlaneChunkClip(newValue)
     );
 
+    useProceduralWater.setTooltip(new Tooltip("Generate customized water ripples using noise."));
```

Comment:
```suggestion
    useProceduralWater.setTooltip(new Tooltip("Generate customized water waves using noise to avoid tiling issues."));
```

Reply:
Maybe we should invert this option, enable it by default and call the non-procedural water "fast waves" or something like that. But that's for a future PR, I guess. The tiling is visible on lots of renders, unfortunately.